contact/contact.html

```diff
@@ -1,11 +1,29 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./contact.css" />
-    <title>contact</title>
-  </head>
-  <body></body>
-</html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/620f3808e1.js" crossorigin="anonymous"></script>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./contact.css">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <!-- <img src="../assets/Vector.png" class="back"> -->
+        <i class="fas fa-arrow-left back" ></i>   
+       <h1 class="covid-19">C<img src="../assets/coviddd.png" class="covid-img">vid-19</h1>
+        <img src="../assets/dark-mode.png" class="dark-mode">
+    </header>
+    <section class="container">
+        <section class="part ">
```

Comment:
Why did you use one section element inside another?
Take a look of this code example
```<header></header>
<section>
	<article>
		<figure>
			<img>
			<figcaption></figcaption>
		</figure>
	</article>
</section>
```